crates/interpreter/src/exec.rs

```diff
@@ -1029,6 +1036,15 @@ impl<'m> Thread<'m> {
         values
     }
 
+    fn only_pop_values(&mut self, n: usize) -> Vec<Val> {
```

Comment:
Just found a minor improvement that I could make later: to unify this function and the `pop_values(&mut self, n: usize)` above, create a `pop_values(&mut self, n: usize, pop_once: fn(&mut self) -> Val)` by passing the function used in `values.push()` as `pop_once`.

Reply:
Done by generics.